nodes/Imap/operations/email/functions/EmailDownload.ts

```diff
@@ -0,0 +1,61 @@
+import { FetchQueryObject, ImapFlow } from "imapflow";
+import { IExecuteFunctions, INodeExecutionData } from "n8n-workflow";
+import { IResourceOperationDef } from "../../../utils/CommonDefinitions";
+import { getMailboxPathFromNodeParameter, parameterSelectMailbox } from "../../../utils/SearchFieldParameters";
+
+export const downloadOperation: IResourceOperationDef = {
+  operation: {
+    name: 'Download',
+    value: 'download',
+  },
+  parameters: [
+    {
+      ...parameterSelectMailbox,
+      description: 'Select the mailbox',
+    },
+    {
+      displayName: 'Email UID',
+      name: 'emailUid',
+      type: 'string',
+      default: '',
+      description: 'UID of the email to download',
+    },
+    {
+      displayName: 'Put Output File in Field',
```

Comment:
I would make an option: output to a **text field** or to a **binary**. This will allow easier chain processing of raw eml content, if needed.

Please let me know if you will implement the switch, or I could implement it later.